gui/system/forms.py

```diff
@@ -2849,6 +2849,19 @@ class CertificateAuthoritySignCSRForm(ModelForm):
         label=(_("CSRs"))
     )
 
+    def clean_cert_name(self):
+        cdata = self.cleaned_data
+        name = cdata.get('cert_name')
+        certs = models.Certificate.objects.get(cert_name=name)
```

Comment:
get() will raise an exception if the certificate does not exist. You will have better luck using `filter` and `exists`